how-to/donate_project.md

```diff
@@ -0,0 +1,111 @@
+# Project Donation Application
+
+Please see the [proposals/new-project-donation-process.md](Documentation) to
+better understand the full process for submitting a new project.
+In short, copy this Application Template and populate the document.
+--!>
+
+## CNCF checklist
```

Comment:
Can we get some feedback from CNCF on this please ?
Do we need to perform any additional actions when we accept new projects under Kubeflow Ecosystem ?
E.g. transfer trademark.
cc @rochaporto @castrojo @kgamanji